src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3770,7 +3859,9 @@ export class StudyViewPageStore {
             this.mutationProfiles.isPending ||
             this.cnaProfiles.isPending ||
             this.structuralVariantProfiles.isPending ||
-            this.survivalClinicalAttributesPrefix.isPending;
+            this.survivalClinicalAttributesPrefix.isPending ||
+            this.sampleTreatments.isPending ||
```

Comment:
this shouldn't be here. it render complete page again whenever a filter is applied as the request for `sampleTreatments` and `patientTreatments` depends on filter.
I think it is also causing https://github.com/cBioPortal/cbioportal/issues/7858